test/test_mps.py

```diff
@@ -5175,6 +5222,30 @@ def helper(shape, dtype=torch.float32):
         for dtype in [torch.float32, torch.int32, torch.int64, torch.bool]:
             helper((2, 3), dtype)
 
+    # torch.cumprod is not supported on mps until macOS 13
+    @unittest.skipIf(product_version < 13.0, "Skipped on macOS 12")
+    def test_prod_backward(self):
```

Comment:
For some reason the `
Mac MPS / macos-12-py3-arm64-mps / test` suite is failing with `test_mps.py::TestNLLLoss::test_prod_backward FAILED` despite that this test should be skipped. Am I misunderstanding the `skipIf` functionality here?